src/Maatwebsite/Excel/Writers/LaravelExcelWriter.php

```diff
@@ -350,6 +365,8 @@ protected function _download(Array $headers = [])
      */
     public function store($ext = 'xls', $path = false, $returnInfo = false)
     {
+        $this->checkExtensionIsValid($ext);
```

Comment:
Moved method call to `_setWriter` [868aea9](https://github.com/Maatwebsite/Laravel-Excel/pull/1129/commits/868aea9c3c699c2c3807575b6f2a38d5e8f4307b#diff-a5f6bef155051f62cca633eb1eff4adaR564)